azurerm/internal/services/network/firewall_policy_resource.go

```diff
@@ -83,10 +83,12 @@ func resourceArmFirewallPolicy() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						// TODO 3.0 - remove this property
 						"network_rule_fqdn_enabled": {
-							Type:     schema.TypeBool,
-							Optional: true,
-							Default:  false,
+							Type:       schema.TypeBool,
+							Optional:   true,
+							Computed:   true,
+							Deprecated: "This property is deprecated since service no longer support it since Nov.2020",
```

Comment:
Could we be more clear here? Also the date may be different for different regions?
```suggestion
							Deprecated: "This property has been deprecated as the service team has removed it from all API versions and is no longer supported by Azure. It will be removed in v3.0 of the provider.",
```